storage/tianmu/core/tianmu_table.cpp

```diff
@@ -604,25 +604,26 @@ void TianmuTable::Field2VC(Field *f, loader::ValueCache &vc, size_t col) {
     case MYSQL_TYPE_LONG:
     case MYSQL_TYPE_INT24:
     case MYSQL_TYPE_LONGLONG: {
-      int64_t v = f->val_int();
-      if (m_attrs[col]->GetIfAutoInc() && v == 0)
+      int64_t value = f->val_int();
+      if (m_attrs[col]->GetIfAutoInc() && value == 0)
         // Value of auto inc column was not assigned by user
         *reinterpret_cast<int64_t *>(vc.Prepare(sizeof(int64_t))) = m_attrs[col]->AutoIncNext();
       else
-        *reinterpret_cast<int64_t *>(vc.Prepare(sizeof(int64_t))) = v;
+        *reinterpret_cast<int64_t *>(vc.Prepare(sizeof(int64_t))) = value;
       vc.ExpectedSize(sizeof(int64_t));
       if (m_attrs[col]->GetIfAutoInc()) {
         // inc counter should be set to value of user assigned
-        if (static_cast<uint64_t>(v) > m_attrs[col]->GetAutoInc()) {
-          m_attrs[col]->SetAutoInc(v);
+        if (static_cast<uint64_t>(value) > m_attrs[col]->GetAutoInc()) {
```

Comment:
and how about this case:
`value` is less than `0`.

Reply:
When `col value` is negative or greater than  INT64_MAX，`value `is less than 0.

Reply:
In addition, `col_value < 0`  will not change  auto_increment value, which is  consistent with innodb